docs/source/install/linux-beta.rst

```diff
@@ -0,0 +1,100 @@
+.. |TexText| replace:: **TexText for Inkscape 1.0 beta**
+.. |Inkscape| replace:: **Inkscape 1.0 beta**
+
+.. role:: bash(code)
+   :language: bash
+   :class: highlight
+
+.. role:: latex(code)
+   :language: latex
+   :class: highlight
+
+.. _linux-beta-install:
+
+==================
+|TexText| on Linux
+==================
+
+This guide explains how to install and use |TexText| together with |Inkscape| on a system
+on which also Inkscape 0.94.x is installed.
+
+Preparation
+===========
+
+1. Backup your existing Inkscape extension directory
+
+    .. code-block:: bash
+
+        cp -r ~/.config/inkscape/extensions ~/.config/inkscape/extensions.backup
+
+2. Remove the old TexText installation:
+
+    .. code-block:: bash
+
+        rm -rf ~/.config/inkscape/extensions/textext
+        rm ~/.config/inkscape/extensions/textext.inx
+
+
+Download and install the |Inkscape| app image file
+==================================================
+
+1. You can download |Inkscape| from https://inkscape.org/release/inkscape-1.0beta1/. Or
+   execute
+
+    .. code-block:: bash
+
+        wget -c https://inkscape.org/gallery/item/14918/Inkscape-fe3e306-x86_64.AppImage
+
+2. Make the downloaded AppImage executable:
+
+    .. code-block:: bash
+
+        chmod u+x Inkscape-fe3e306-x86_64.AppImage
+
+3. Extract the AppImage:
+
+    .. code-block:: bash
+
+        ./Inkscape-fe3e306-x86_64.AppImage --appimage-extract
+
+After this operation |Inkscape| will reside in ``~/squashfs-root/bin``. You can test it by
```

Comment:
Doesn't it extract to current folder? Do you assume here that CWD in $HOME?

Reply:
Good catch. It is extracted into the current directory. Thanks! I update the doc.